stylesheets/partials/features/_card.scss

```diff
@@ -1,110 +1,110 @@
-.cards {
-	margin: 1em 0 2em;
-	float: left;
-	width: 100%;
+// .cards {
```

Comment:
Again - commented out on purpose?

Reply:
yes, there is a card component that is different from bcpl, but i plan on using